.dev/docker/env/Dockerfile

```diff
@@ -2,10 +2,8 @@
 # https://gitlab.com/nvidia/container-images/cuda/tree/master/dist
 # To enable cuda, use "--build-arg USE_CUDA=true" during image build process
 ARG USE_CUDA
-ARG CUDA_VERSION="9.0"
+ARG CUDA_VERSION="9.2"
```

Comment:
Any benefits to changing this?

Reply:
Maybe not although I believe I'm the one who suggested the change on Discord. I recall there were some compilation issues between 9.0 and 9.2 and I would assume people would pick the highest minor version when picking CUDA 9. That's also the only one in the 9.X releases that PyTorch is supporting.